auth-service/src/main/java/com/petclinic/authservice/businesslayer/UserService.java

```diff
@@ -63,4 +63,5 @@ public interface UserService {
     void updatePassword(String newPassword, String token);
 
     void processResetPassword(UserResetPwdWithTokenRequestModel resetRequest);
+    UserPasswordLessDTO updateUserRole(String id, RolesChangeRequestDTO roles, String token);
```

Comment:
I think should specify the type of id by using userId or simply id since they are quite different.